_package/xmscore/misc/observer.py

```diff
@@ -0,0 +1,26 @@
+from .._xmscore.misc import Observer as Obs
+
+
+class Observer(Obs):
+    """Mock Observer class for testing."""
+
+    def __init__(self, instance=None):
```

Comment:
If not using instance, remove the argument.

Reply:
fixed in commit f6fbbec